### DISCUSSION TITLE
Publish resource(s) option in plugin

### DISCUSSION THREAD

**Discussion by thethunderturner** [filamentphp/filament#15199] [Ideas]
Is it possible that we add an option in the future where we can publish resources for plugins?

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/support/style-customization#publishing-blade-views

**Reply by thethunderturner** [reply]
Ah sorry, perhaps I should've been more clear. I actually meant filament resources (and their pages), not blade views

**Reply by leandrocfe** [reply]
publish Filament resources? What does it mean?

**Reply by thethunderturner** [reply]
As in lets say you have a resource for your [plugin](https://filamentphp.com/docs/3.x/panels/resources/getting-started). If you publish that resource, then you can pretty much override it (just how we do with blade views,config etc)
